Dfe.Academies.Academisation.Domain/ProjectAggregate/Project.cs

```diff
@@ -70,6 +71,57 @@ public static CreateResult Create(IApplication application)
 		return new CreateSuccessResult<IProject>(new Project(projectDetails));
 	}
 
+	public static CreateResult CreateInvoluntaryProject(InvoluntaryProject project)
+	{
+		// Validate 
+		if (project.JoinTrust == null)
+		{
+			return new CreateValidationErrorResult(new List<ValidationError>
+			{
+				new("Join Trust", "Join Trust in the model must not be null")
+			});
+		}
+		if (project.Schools == null || project.Schools.Any() is false)
+		{
+			return new CreateValidationErrorResult(new List<ValidationError>
+			{
+				new("Schools", "Schools in the model must not be null")
+			});
+		}
+
+		// Get school 
```

Comment:
Small thing but I'm not sure this comment adds a great deal.

Reply:
removed